Sources/Misc/Concurrency/OperationDispatcher.swift

```diff
@@ -108,6 +109,8 @@ private extension Delay {
         case .none: return 0
         case .`default`: return 0
         case .long: return Self.maxJitter
+        case .timeInterval(let timeInterval):
```

Comment:
if we don't need the jitter, we might as well make another method in operation dispatcher that just takes an exact value as `TimeInterval` rather as an overload to the one we're using instead of passing in a min and max value that are the same

Reply:
I think the argument can be made either way. If we were to make that change, we'd then have two similar dispatch functions that happen after a period of time: 
- `func dispatchOnWorkerThread(delay: Delay)`
- `func dispatchOnWorkerThread(after timeInterval: TimeInterval)`

When I read these function signatures, it isn't clear which one I should choose because the function signatures signify the same thing. 

If we were to go through with this change, I'd recommend renaming `Delay` to something like `JitterableDelay` to indicate that one of these APIs includes a jitter and the other doesn't. I'm down to make this change - any objections to that renaming?